cmd/sharded-test-server/frontproxy.go

```diff
@@ -53,9 +60,9 @@ func startFrontProxy(ctx context.Context, args []string, servingCA *crypto.CA, h
 		lineprefix.Color(color.New(color.FgHiWhite)),
 	)
 
-	if err := ioutil.WriteFile(filepath.Join(workDirPath, ".kcp-front-proxy/mapping.yaml"), []byte(`
+	if err := ioutil.WriteFile(filepath.Join(workDirPath, ".kcp-front-proxy/mapping.yaml"), []byte(fmt.Sprintf(`
```

Comment:
Since we're now print-formatting some YAML, would it be difficult to define an anonymous type here and marshal it instead?

Reply:
Will look into it